api/spaces/spaces.go

```diff
@@ -81,6 +81,24 @@ func (api *API) CreateSpace(name string, cidrs []string, public bool) error {
 	return response.OneError()
 }
 
+// ShowSpace shows details about a space.
+// Containing subnets, applications and machines count associated with it.
+func (api *API) ShowSpace(name string) (params.ShowSpaceResults, error) {
+	var response params.ShowSpaceResults
+	var args interface{}
+	args = params.Entities{
+		Entities: []params.Entity{{Tag: names.NewSpaceTag(name).Id()}},
```

Comment:
`String()` instead of `Id()`

Reply:
Wouldn't that make e.g. it to: `space-<supplied_space_name>`or is that what you meant?

Reply:
ahh, its wanted, just looked into the `ParseTag` code